static/styles/zulip.css

```diff
@@ -418,16 +398,13 @@ a:hover code {
     white-space: nowrap;
 }
 
-.sidebar-title,
-.share-the-love-title {
+.sidebar-title {
     font-size: 11px;
     font-weight: normal;
     display: inline;
 }
```

Comment:
I think we should just remove `share-the-love-title` and use `sidebar-title` instead; there's no need for an extra class for this.